recipes/rmlui/3.3/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RmluiConan(ConanFile):
+    name = "rmlui"
+    description = "RmlUi - The HTML/CSS User Interface Library Evolved"
+    homepage = "https://github.com/mikke89/RmlUi"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("conan", "css", "gui", "html", "lua", "rmlui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "build_lua_bindings": [True, False],
+        "disable_rtti_and_exceptions": [True, False],
+        "enable_precompiled_headers": [True, False],
+        "enable_tracy_profiling": [True, False],
```

Comment:
Is this a usable option for a user of `rmlui`?
If this is an option for the developers of `rmlui` to profile their own library, then it might be not useful for users of the conan `rmlui` package.

Reply:
Well.. I believe that there never is a clear-cut answer to this in case of performance profiling support. However, as I've come to understand, the Tracy library (https://github.com/wolfpld/tracy) is neither available through conan, nor included in the library. Thus, supporting it not possible at this point, so I'll just remove it for now. Thanks for challenging it!